game/game.css

```diff
@@ -215,17 +215,21 @@ table{
     transition-duration: 300ms;
  }
 
+ #game--life--img, #game--life--level {
+    display: inline;
+ }
+
 .apple{
-    background-image: url(tiles/applee.png),url(tiles/floor.png);
+    background-image: url(./img/apple.png),url(tiles/floor.png);
```

Comment:
floor zostalo w `tiles`? a czemu apple poszlo sobie gdzie indziej? chyba floor tez powinno byc przeniesione?

Reply:
floor (with variations) and wall (with variations) constitute map tiles - in tiles folder.
apple, chest, monster and player are other images (in img folder), displated on/over map tiles.

this specific piece of code displays apple img over floor tile.